backend/dissemination/views.py

```diff
@@ -225,10 +279,35 @@ def get(self, request, report_id, file_type):
         return redirect(get_download_url(filename))
 
 
-class SummaryReportDownloadView(ReportAccessRequiredMixin, View):
+class SingleSummaryReportDownloadView(ReportAccessRequiredMixin, View):
     def get(self, request, report_id):
+        """
+        Given a report_id in the URL, generate the summary report in S3 and
+        redirect to its download link.
+        """
         sac = get_object_or_404(General, report_id=report_id)
         filename = generate_summary_report([sac.report_id])
         download_url = get_download_url(filename)
 
         return redirect(download_url)
+
+
+class MultipleSummaryReportDownloadView(View):
+    def post(self, request):
+        """
+        1. Get the report_ids from a fresh search with the provided search parameters
+        2. Generate a summary report with the report_ids, that goes into S3
+        3. Redirect to the download url of this new report
+        """
+        form = SearchForm(request.POST)
+
+        cleaned_data = clean_form_data(form)
+        include_private = include_private_results(request)
+        results = run_search_general(cleaned_data, include_private)
+        results = results[:1000]  # Hard limit XLSX downloads to 1000 records
```

Comment:
can we wrap this 1000 in a const (e.g. `SUMMARY_REPORT_MAX`) that we use here and pass into the `search.html` context for deciding whether or not to enable the download button, so that we can adjust that number in a single place?